src/ActionMenu/ActionMenu.tsx

```diff
@@ -46,8 +46,13 @@ const Menu: React.FC<React.PropsWithChildren<ActionMenuProps>> = ({
   const onOpen = React.useCallback(() => setCombinedOpenState(true), [setCombinedOpenState])
   const onClose = React.useCallback(() => setCombinedOpenState(false), [setCombinedOpenState])
 
+  const menuButtonChild = React.Children.toArray(children).find(
+    child => React.isValidElement<ActionMenuButtonProps>(child) && child.type === MenuButton,
```

Comment:
should we also check for `child.type === Anchor`?

https://github.com/primer/react/pull/3897/files#diff-887b5dc58d065fbd8f80bac1791c6134698a717df97e9c338f32d1b3b72fb8d0L69

Reply:
Yea, probably. Will update now.